v/vault/Dockerfiles/1.5.3-ubi8/Dockerfile

```diff
@@ -15,8 +15,8 @@ ENV PATH $PATH:$GOPATH/bin
 
 # Set up our base tools, and Vault.
 RUN set -eux; \
-    yum install -y openssl sudo \ 
-    make git gcc wget && \
+    yum install -y https://www.rpmfind.net/linux/centos/8-stream/BaseOS/ppc64le/os/Packages/libgomp-8.4.1-1.el8.ppc64le.rpm \
```

Comment:
Can you please explain the requirement of `libgomp`?

Reply:
libgomp-8.4.1 is required by gcc 8.4.1. The yum repository was installing lower version of libgomp and hence the gcc installation was failing. 
I believe, this change is no more required. Looks like yum repo has the upgraded version now. Have reverted this change.